frontend/src/app/core/path-helper/apiv3-paths.ts

```diff
@@ -43,8 +43,6 @@ export class ApiV3Paths {
     filters.add('member', '=', [projectId.toString()]);
     // That are users:
     filters.add('type', '=', ['User', 'Group']);
-    // That are not the current user:
-    filters.add('id', '!', ['me']);
 
     if (term && term.length > 0) {
```

Comment:
I fear that other parts of the code will start using the `principals` path eventually and then `me` showing up will not fit the use case. Then somebody might readd the constraint.

We can keep it the way it is but to me, that is a strong indicator that a feature spec is necessary here. Otherwise, we will not be able to guarantee the functionality.

Reply:
Done. Could you check the feature spec, please? I saw a notable behavior. I added a block for testing "myself mentioning". First I added it before "user mentioning" but that didn't work. So I swapped both blocks and then it worked.

Can you reproduce this behavior?

Reply:
@ulferts Could you already have had a look at this?